SilMock/Google/Service/Directory/Resource/GroupsAliases.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace SilMock\Google\Service\Directory\Resource;
+
+use Exception;
+use Google\Service\Directory\Alias as GoogleDirectory_GroupsAlias;
+use Google\Service\Directory\Aliases as GoogleDirectory_GroupsAliases;
+use SilMock\Google\Service\DbClass;
+use SilMock\Google\Service\Directory\ObjectUtils;
+
+class GroupsAliases extends DbClass
+{
+    public function __construct(?string $dbFile = null)
+    {
+        parent::__construct($dbFile, 'directory', 'groupsaliases');
+    }
+
+    public function delete(string $groupKey, string $alias, array $optParams = [])
+    {
+        $groupRecords = $this->getRecords();
```

Comment:
Minor nitpick: I think it might be easier to use the same name for variables that contain the same thing in various functions. `$this->getRecords()` is stored in the following variables in various functions:
- $groupRecords
- $directoryGroupAliasRecords

Reply:
I'm not following.

Reply:
Oh, got it.